test/JDBC/input/datatypes/TestSQLVariant-vu-prepare.sql

```diff
@@ -261,3 +261,11 @@ Insert into testsqlvariant_sourceTable22 values (cast ('2016-10-23 12:45:37.123'
 go
 Insert into testsqlvariant_sourceTable22 values (cast ('2016-10-23 12:45:37.12345' as datetime2(5)), cast ('2016-10-23 12:45:37.123456' as datetime2(6)));
 go
+
+-- Test sql_variant cast (Test for BABEL-3404)
+CREATE VIEW testsqlvariant_view1 AS SELECT CAST('abc' AS sql_variant) AS col;
```

Comment:
object name should be filename_xxx

Reply:
Are you suggesting that we should also include `-vu-prepare` part in the object name?

Reply:
yes, whatever is in the file name should be the prefix for object name.

Reply:
Updated the object name!